packages/server/utils/OpenAIServerManager.ts

```diff
@@ -34,6 +34,36 @@ class OpenAIServerManager {
     })
   }
 
+  async getReflectionGroupTitle(reflections: Reflection[]) {
+    if (!this.openAIApi) return null
+    const prompt = `Below are lines of texts possibly talking about the same topic. Summarize all of the content in less than 3 simple words, with a leading emoji if possible. If you cannot generate a meaningful summary title, simply return "NO_RESPONSE" with nothing else instead. You will be penalized if your return exceeds 3 words of length, or the language of your output does not match the majority language of the input.
```

Comment:
+1 have you seen prompts written using "penalized" before? I'd say it'd be a clearer instruction to just say "do not exceed 3 words"

Reply:
This prompt has been "polished" by ChatGPT itself 😂

Reply:
Isn't there an off-by-one error in the prompt?
> less than 3 simple words

vs.
> exceeds 3 words of length